crates/uv/tests/pip_install.rs

```diff
@@ -859,11 +795,9 @@ fn install_editable_and_registry() -> Result<()> {
     );
 
     // Install the editable version of Black. This should remove the registry-based version.
-    uv_snapshot!(filters, command(&context)
+    uv_snapshot!(context.filters(), command(&context)
         .arg("-e")
-        .arg("../../scripts/packages/black_editable")
-        .current_dir(&current_dir)
-        .env("CARGO_TARGET_DIR", "../../../target/target_install_editable"), @r###"
```

Comment:
Lots of tests set the `CARGO_TARGET_DIR`, presumably for editables that used to use Maturin? I'm actually quite confused but I've removed it from many test cases as it appears superfluous.